tools/templates/Makefile-backend-common.in

```diff
@@ -126,7 +136,7 @@
 @backend_prefix@-stresstest: @backend_prefix@-stresstest@bpm(HARNESS_TYPE)@
 
 @backend_prefix@-coretest5: @backend_prefix@-all
-	@bpm(HARNESS5)@ @nfpl(t/01-sanity t/02-rakudo t/04-nativecall t/05-messages t/06-telemetry t/07-pod-to-text t/08-performance t/09-moar t/10-qast)@
+	@bpm(HARNESS5)@ @nfpl(@bpm(BACKEND_INDEPENDENT_TEST_FILES)@)@ @nfpl(@bpm(BACKEND_SPECIFIC_TEST_FILES)@)@
```

Comment:
`@nfpl()@` doesn't make any sense here because what it gets as input is `$(<backend_prefix>_BACKEND_SPECIFIC_TEST_FILES)`, i.e. a Makefile variable. See the comment above how to get it fixed.